create-config.sh

```diff
@@ -18,6 +18,7 @@ docker context create "opam.ci.ocaml.org" --description "OCaml - Opam Repo CI" -
 docker context create "ocaml.ci.dev" --description "Tarides - OCaml CI" --docker "host=ssh://root@ocaml.ci.dev"
 docker context create "check.ci.ocaml.org" --description "OCaml - Opam Health Check" --docker "host=ssh://root@check.ci.ocaml.org"
 docker context create "registry.ci.dev" --description "Docker Registry" --docker "host=ssh://root@registry.ci.dev"
+docker context create "download.dune.ci.dev" --description "Dune Binary Distribution" --docker "host=ssh://root@download.dune.ci.dev"
```

Comment:
```suggestion
docker context create "preview.dune.build" --description "Dune Binary Distribution" --docker "host=ssh://root@preview.dune.build"
```

Reply:
I think we should use get.dune.build here to avoid the need to change it in the future.